core/src/main/scala/cell/Cell.scala

```diff
@@ -5,13 +5,14 @@ import java.util.concurrent.{ CountDownLatch, ExecutionException }
 
 import scala.annotation.tailrec
 
-import scala.concurrent.{ ExecutionContext, OnCompleteRunnable }
+import scala.concurrent.OnCompleteRunnable
 import scala.util.control.NonFatal
 import scala.util.{ Failure, Success, Try }
 
 import lattice.{ Lattice, LatticeViolationException, Key, DefaultKey }
 
 trait Cell[K <: Key[V], V] {
+  val completer: CellCompleter[K, V]
```

Comment:
I believe this should be `private[cell]`, since `Cell[K,V]` is supposed to be read-only.